src/main/webapp/app/shared/layouts/main/main.component.html

```diff
@@ -1,5 +1,11 @@
 <jhi-alert-overlay />
-<div id="page-wrapper" class="page-wrapper" [ngClass]="{ 'footer-height-dev': !isProduction || isTestServer }" cdkScrollable>
+<div
+    id="page-wrapper"
+    class="page-wrapper"
+    [ngClass]="{ 'footer-height-dev': (!isProduction || isTestServer) && !isExamStarted }"
+    [ngStyle]="{ 'overflow-y': isTestRunExam ? 'hidden' : 'auto' }"
```

Comment:
What about test exams specifically requires us to hide the overflow of the page?

Reply:
This is actually not for test exams, but test runs for the instructors. Since the exam layout is changed and we use sidebar and sidebar content during the exam, overflow of the page should get hidden for instructors as well. In the test runs, there are also other page components during the exam for instructors, that's why I needed to use another approach for test runs.